addons/web_editor/static/src/js/editor/snippets.options.js

```diff
@@ -2198,8 +2201,20 @@ const ListUserValueWidget = UserValueWidget.extend({
     /**
      * @private
      */
-    _onListItemChange() {
-        this._notifyCurrentState();
+    _onListItemBlur(ev) {
+        if (!this.el.contains(ev.relatedTarget)) {
+            // Update the target and the editor.
+            this._notifyCurrentState(false);
```

Comment:
Same comment as before: dubious comment as it relies on what the widget is used for. I would just not add a comment here.